frontend/src/scenes/persons-management/PersonsManagementScene.test.tsx

```diff
@@ -0,0 +1,127 @@
+import '@testing-library/jest-dom'
+
+import { cleanup, render, screen, waitFor, within } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { Provider } from 'kea'
+import { router } from 'kea-router'
+import { PersonsManagementScene } from 'scenes/persons-management/PersonsManagementScene'
+import { urls } from 'scenes/urls'
+
+import { useMocks } from '~/mocks/jest'
+import { initKeaTests } from '~/test/init'
+
+const testExampleResult = [
+    {
+        id: 1,
+        name: 'Test User',
+        distinct_ids: ['test_id'],
+        properties: {
+            email: 'test@example.com',
+        },
+        created_at: '2021-01-01T00:00:00Z',
+    },
+]
+
+describe('PersonsManagementScene', () => {
+    afterEach(() => {
+        cleanup()
+    })
+
+    beforeEach(() => {
+        useMocks({
+            post: {
+                '/api/environments/:team_id/query': async (req, res, ctx) => {
+                    const payload = await req.json()
+                    if (payload.query.search) {
+                        if (payload.query.search !== 'test@example.com') {
+                            return res(
+                                ctx.json({
+                                    results: [],
+                                    count: 0,
+                                    complete: true,
+                                })
+                            )
+                        }
+                        return res(
+                            ctx.json({
+                                results: [testExampleResult],
+                                count: 1,
+                                complete: true,
+                            })
+                        )
+                    }
+                    // if (payload.query.kind !== 'ActorsQuery') {
+                    //     return res(ctx.json({
+                    //         results: [], count: 0,
+                    //         complete: true,
+                    //     }))
+                    // }
+                    // by default we have more than one actor
```

Comment:
Can this commented code be removed?

Reply:
yep, i'll be lazy and not do that right now in case i can get this through CI so others can copy 🙈 

needs a few more tests here before we could delete cypress tests for the page